src/applications/check-in/day-of/pages/Confirmation/index.jsx

```diff
@@ -21,6 +40,37 @@ const Confirmation = () => {
     selectConfirmationData,
   );
 
+  useEffect(
+    () => {
+      if (
+        !isDayOfDemographicsFlagsEnabled ||
+        demographicsFlagsSent ||
+        getDemographicsConfirmed(window)
```

Comment:
Is `getDemographicsConfirmed` redundant? I'm only seeing it used here. Not sure if this is a leftover or I'm missing something 😅

Reply:
These are both intentional, but looking back I think these could have been named much better. 

`demographicsFlagsSent` is a react state boolean that is there to keep the api call from running more than once when a page loads with that useEffect hook. 

`getDemographicsConfirmed()` pulls the confirmed status from the session storage and will prevent making that api call multiple time after a veteran has checked into an appointment and confirmed their demographics.

Looking back at the See Staff page, I think we may actually need to add it there too. That would prevent a user from checking into their appointment which confirms demographics and then going back and indicating their demographics are not correct. 🤔